link/res/values/strings.xml

```diff
@@ -6,4 +6,8 @@
     <string name="sign_up_message">Pay faster at %1$s and thousands of merchants.</string>
     <string name="sign_up_terms">You agree to Link terms of service and to create a Link with Stripe account.</string>
     <string name="sign_up">Join Link</string>
+
+    <string name="verification_header">Enter your verification code</string>
+    <string name="verification_message">Enter the code sent to %1$s to securely use your saved information.</string>
+    <string name="verification_resend">Resend code</string>
```

Comment:
Should we have a totranslate.xml file here, similar to what we have in other modules?